src/main/java/rayshawn/chatbot/parser/Parser.java

```diff
@@ -0,0 +1,197 @@
+package rayshawn.chatbot.parser;
+
+import static rayshawn.chatbot.messages.Messages.INVALID_COMMAND_FORMAT_MESSAGE;
+import static rayshawn.chatbot.messages.Messages.INVALID_TASK_NUMBER_MESSAGE;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
```

Comment:
Good use of the java.util.regex package, it has helped to improve code readability and maintainability.